src/components/FacilitiesPageViewMore/DeleteFacility.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+    useDisclosure,
+    Button,
+    ButtonGroup,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalCloseButton,
+    ModalBody,
+    ModalFooter,
+} from '@chakra-ui/react';
+import { useBackend } from '../../contexts/BackendContext';
+
+
+function DeleteFacility(props) {
+    const { isOpen, onOpen, onClose } = useDisclosure();
+    const { backend } = useBackend();
+    function ShowModal({ isOpen, onClose }) {
```

Comment:
🚫 **[eslint]** <[no-shadow](https://eslint.org/docs/rules/no-shadow)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'isOpen' is already declared in the upper scope on line 18 column 13.